examples/foo-missionmodel/src/test/java/gov/nasa/jpl/aerie/foomissionmodel/FooActivityTest.java

```diff
@@ -23,27 +22,21 @@
 import static org.assertj.core.api.Assertions.assertThatNoException;
 import static org.assertj.core.api.Assertions.within;
 
+// The `@ExtendWith` annotation injects the given extension into JUnit's testing apparatus.
+// Our `MerlinExtension` hooks test class construction and test method execution,
+//   executing each with the appropriate simulation context.
+@ExtendWith(MerlinExtension.class)
 @TestInstance(Lifecycle.PER_CLASS)
 public final class FooActivityTest {
-
-  // The `@RegisterExtension` annotation programmatically injects the given extension into JUnit's testing apparatus.
-  // Our `MerlinExtension` hooks test class construction and test method execution,
-  //   executing each with the appropriate simulation context.
-  @RegisterExtension
-  public static final MerlinExtension<ActivityTypes, Mission> ext = new MerlinExtension<>();
-
   private final Mission model;
 
   // Initializers and the test class constructor are executed in an "initialization" Merlin context.
   // This means that models can be created (and cell storage allocated, and daemons spawned),
   //   but simulation control actions like `waitFor`, `delay`, and `emit` cannot be performed.
-  // The `MerlinTestContext` does not need to be declared as a parameter, but will be injected if declared.
-  public FooActivityTest(final MerlinTestContext<ActivityTypes, Mission> ctx) {
+  // The `Registrar` does not need to be declared as a parameter, but will be injected if declared.
+  public FooActivityTest(final Registrar registrar) {
```

Comment:
This change also means all tests consistently use `@ExtendWith` instead of sometimes needing to use `RegisterExtension`. I consider that an ergonomics win too :)